types/react-google-tag-manager/index.d.ts

```diff
@@ -0,0 +1,18 @@
+// Type definitions for react-google-tag-manager 2.2
+// Project: https://github.com/gaearon/react-google-tag-manager
+// Definitions by: Gianluca Venturini <https://github.com/gianluca-venturini>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.6
+
+export default function GTMParts(args: {
+    id: string,
+    dataLayerName?: string,
+    additionalEvents?: {},
```

Comment:
I don't think `{}` is the correct type. From the documentation [here](https://www.npmjs.com/package/react-google-tag-manager#how-to-use), and [here](https://developers.google.com/tag-manager/devguide#events), it looks like this should be `Record<string, string>` (or just `{ [key: string]: string; }`).